lib/dialects/abstract/query-generator.js

```diff
@@ -85,90 +57,19 @@ const QueryGenerator = {
     return 'DESCRIBE ' + table + ';';
   },
 
-  /*
-    Returns a query for dropping a table.
-  */
   dropTableQuery(tableName, options) {
     options = options || {};
 
     return `DROP TABLE IF EXISTS ${this.quoteTable(tableName)};`;
   },
 
-  /*
-    Returns a rename table query.
-    Parameters:
-      - originalTableName: Name of the table before execution.
-      - futureTableName: Name of the table after execution.
-  */
   renameTableQuery(before, after) {
     return `ALTER TABLE ${this.quoteTable(before)} RENAME TO ${this.quoteTable(after)};`;
   },
 
   /*
-    Returns a query, which gets all available table names in the database.
-  */
-  /* istanbul ignore next */
-  showTablesQuery() {
-    throwMethodUndefined('showTablesQuery');
-  },
-
-  /*
-    Returns a query, which adds an attribute to an existing table.
-    Parameters:
-      - tableName: Name of the existing table.
-      - attributes: A hash with attribute-attributeOptions-pairs.
-        - key: attributeName
-        - value: A hash with attribute specific options:
-          - type: DataType
-          - defaultValue: A String with the default value
-          - allowNull: Boolean
-  */
-  /* istanbul ignore next */
-  addColumnQuery(tableName, attributes) {
-    throwMethodUndefined('addColumnQuery');
-  },
-
-  /*
-    Returns a query, which removes an attribute from an existing table.
-    Parameters:
-      - tableName: Name of the existing table
-      - attributeName: Name of the obsolete attribute.
-  */
-  /* istanbul ignore next */
-  removeColumnQuery(tableName, attributeName) {
-    throwMethodUndefined('removeColumnQuery');
-  },
-
-  /*
-    Returns a query, which modifies an existing attribute from a table.
-    Parameters:
-      - tableName: Name of the existing table.
-      - attributes: A hash with attribute-attributeOptions-pairs.
-        - key: attributeName
-        - value: A hash with attribute specific options:
-          - type: DataType
-          - defaultValue: A String with the default value
-          - allowNull: Boolean
-  */
-  /* istanbul ignore next */
-  changeColumnQuery(tableName, attributes) {
-    throwMethodUndefined('changeColumnQuery');
-  },
-
-  /*
-    Returns a query, which renames an existing attribute.
-    Parameters:
-      - tableName: Name of an existing table.
-      - attrNameBefore: The name of the attribute, which shall be renamed.
-      - attrNameAfter: The name of the attribute, after renaming.
-  */
-  /* istanbul ignore next */
-  renameColumnQuery(tableName, attrNameBefore, attrNameAfter) {
-    throwMethodUndefined('renameColumnQuery');
-  },
-
-  /*
     Returns an insert into command. Parameters: table name + hash of attribute-value-pairs.
+   @private
```

Comment:
👍 for having more documentation of private methods, but marking them with `@private`